src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ul class="calendar calendar--start-day-mon calendar--month-length-31">
+      <li class="calendar__item"></li>
```

Comment:
Try to rename class for this test 
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/28e5cf85-73ac-48f3-a086-994c134b20f8)
```suggestion
      <li class="calendar__day"></li>
```